charts/document-engine/values.simple.yaml

```diff
@@ -1,43 +1,48 @@
-pspdfkit:
-  license:
-    activationKey: ""
-    externalSecret: 
-      name: ""
-      key: DOCUMENT_ENGINE_ACTIVATION_KEY
-  auth:
-    api:
-      apiToken: secret
-    dashboard:
-      username: admin
-      password: admin
-  log:
-    level: debug
+documentEngineLicense:
+  activationKey: ""
+  externalSecret: 
+    name: ""
+    key: DOCUMENT_ENGINE_ACTIVATION_KEY
+
+apiAuth:
+  apiToken: secret
+
+config:
   trustedProxies: default
-  signingService:
+
+documentSigningService:
+  enabled: true
+  url: http://signing-service:6000/sign
+  timeoutSeconds: 10
+
+assetStorage:
+  databaseMigrationJob: 
+    enabled: false
+  databaseEngine: postgresql
```

Comment:
We always need DB engine if you work with DE in "Server/with storage mode", it does not necessarily relate to asset storage. What about splitting it? e.g.:

```
# Config related to the main DE database.
database: 
  databaseMigrationJob: 
  enabled: false
  databaseEngine: postgres
  # I prefer the same key here as in the `databaseEngine` above
  postgres: 
     ...

# Config related to asset storage.
assetStorage:
   assetSotrageBackend: built-in
   enableAssetStorageFallback: false
```

Reply:
This is exactly what I thought about. Hesitated. Can still do.